tracee-ebpf/tracee/co-re/vmlinux.h

```diff
@@ -705,8 +705,10 @@ struct tcphdr {
 };
 
 struct udphdr {
-	__be16 source;
-	__be16 dest;
+	__be16	source;
+    __be16	dest;
+    __be16	len;
```

Comment:
are these fields used in this PR?

Reply:
nope i think its due the dns_eent's ill remove it